### PR TITLE
Features/binary transport for PyLECO

### LINKED ISSUE #308: PyLECO binary transport
In https://github.com/pymeasure/pyleco/pull/82 I add native support for binary data transport via PyLECO.

I invite you to leave your feedback regarding how it should be implemented for usage.

### PR DESCRIPTION
**Pull request by BenediktBurger**
Follow up of #319 (now for Pymodaq 4.4) , closes #308

### CODE REVIEW

File: src/pymodaq/utils/leco/daq_move_LECODirector.py [hunk 1]
```diff
@@ -143,22 +146,29 @@ def stop_motion(self) -> None:
...
+        self, position: Union[str, float, None], additional_payload=None
+    ) -> DataActuator:
+        if position:
+            if isinstance(position, str):
+                deserializer = DeSerializer.from_b64_string(position)
+                pos = deserializer.dwa_deserialization()
+            else:
+                pos = DataActuator(data=position)
+        elif additional_payload is not None:
+            pos = DeSerializer(additional_payload[0]).dwa_deserialization()
```
Comment (seb5g) [inline src/pymodaq/utils/leco/daq_move_LECODirector.py:159]: what would be the additional payload?

**Review by seb5g — APPROVED** [review]
sound code

**Comment by seb5g** [comment]
Will you explain a bit the inner mechanisms at the pymodaq days?

File: src/pymodaq/utils/leco/daq_move_LECODirector.py (hunk 1 above)
Comment (BenediktBurger) [reply src/pymodaq/utils/leco/daq_move_LECODirector.py:159]: How the binary transfer works:
The first payload frame contains the JSON response.
The second (and further) frames contain binary objects. These are the `additional_payload`.

Therfore, the sending party sends the binary version of the PyMoDAQ object in the first additional payload frame and here it gets deserialized.

**Comment by BenediktBurger** [comment]
Yes, I will.

**Comment by seb5g** [comment]
Then I propose we discuss further implementation after (or during) your talk. By the way I have trouble really undersanding the vocabulary payload and frame. Maybe you should explicit this in your talk

**Pull request merged**